config/locales/javascript/javascript.en.yml

```diff
@@ -300,5 +300,8 @@ en:
       count:
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        one: "1 vote"
```

Comment:
Other translations in that file look like this:
```
reshares:
  zero: "<%= count %> Reshares"
  one: "<%= count %> Reshare"
  other: "<%= count %> Reshares"
```

I'm not familiar with our translation code but I'd try to do the same here.